apps/wallet/src/shared/cryptography/keystore.ts

```diff
@@ -5,8 +5,20 @@ import {
 	encrypt as metamaskEncrypt,
 	decrypt as metamaskDecrypt,
 } from '@metamask/browser-passworder';
+import { randomBytes } from '@noble/hashes/utils';
 
-type Serializable = string | number | boolean | { [index: string]: Serializable } | Serializable[];
+// we use this password + a random one for each time we store the encrypted
+// vault to session storage
+const PASSWORD =
+	process.env.WALLET_KEYRING_PASSWORD ||
+	'344c6f7d04a65c24f35f5c710b0e91e2f2e2f88c038562622d5602019b937bc2c2aa2821e65cc94775fe5acf2fee240d38f1abbbe00b0e6682646a4ce10e908e';
```

Comment:
You mean the constant? or in general how we use the `PASSWORD`?

the constant is there just to make sure we have a password in case process.env.WALLET_KEYRING_PASSWORD is empty

and we use this to encrypt values in session storage with a known password which changes between releases (to make it a bit more difficult to decrypt the values if someone gets access to them)